account_asset_management_menu/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 ForgeFlow, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Assets Management Menu",
+    "version": "15.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": ["account_asset_management", "account_menu"],
```

Comment:
```suggestion
    "depends": ["account_asset_management", "account_usability"],
```
As you can see in https://github.com/OCA/account-financial-tools/pull/1275/, account_menu has been renamed to account_usability

Reply:
@mariadforgeflow I added the dependency